samples/samples/conftest.py

```diff
@@ -240,8 +240,7 @@ def database_ddl():
     return []
 
 
-@pytest.fixture(scope="module")
-def sample_database(
+def create_sample_database(
```

Comment:
Why did we make this change in sample_database?

Reply:
AFAIU, we are creating a helper method to be reused across other methods for creating a database.
PLMK if there is some other reasoning for this change.

Reply:
exactly. existing `sample_database` is a fixture with regional-instance as its argument. We need multi-region instance(nam3) to test our feature. Thus, refactored.